Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/PayPalTokenizationViewModel.swift

```diff
@@ -53,7 +53,7 @@ class PayPalTokenizationViewModel: PaymentMethodTokenizationViewModel {
     override func start() {
         self.didPresentExternalView = { [weak self] in
             if let strongSelf = self {
-                PrimerDelegateProxy.primerHeadlessUniversalCheckoutPaymentMethodDidShow(for: strongSelf.config.type)
```

Comment:
Not mentioned in the migration guide: is that a breaking change?
https://www.notion.so/primerio/Headless-Checkout-Migration-Guide-iOS-3e9e8ebfa8e2402bbf7ffcf85ec16131